doc/source/index.rst

```diff
@@ -272,31 +271,26 @@ Getting Involved
    rllib-offline.rst
    rllib-concepts.rst
    rllib-examples.rst
-   rllib-dev.rst
    rllib-package-ref.rst
+   rllib-dev.rst
 
 .. toctree::
    :maxdepth: -1
-   :caption: RaySGD
+   :caption: Ray SGD
 
    raysgd/raysgd.rst
    raysgd/raysgd_pytorch.rst
    raysgd/raysgd_tensorflow.rst
-   raysgd/raysgd_ref.rst
```

Comment:
Can we also keep the sgd package ref? It currently doesn't show up anywhere else.